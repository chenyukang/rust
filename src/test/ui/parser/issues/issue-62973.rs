// ignore-tidy-trailing-newlines
// error-pattern: aborting due to 5 previous errors

fn main() {}

fn p() { match s { v, E { [) {) }


