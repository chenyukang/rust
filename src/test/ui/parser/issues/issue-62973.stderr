error: mismatched closing delimiter: `)`
  --> $DIR/issue-62973.rs:6:28
   |
LL | fn p() { match s { v, E { [) {) }
   |                           -^ mismatched closing delimiter
   |                           |
   |                           unmatched opening delimiter begins here

error: mismatched closing delimiter: `)`
  --> $DIR/issue-62973.rs:6:31
   |
LL | fn p() { match s { v, E { [) {) }
   |                              -^ mismatched closing delimiter
   |                              |
   |                              unmatched opening delimiter begins here

error: expected one of `,`, `:`, or `}`, found `{`
  --> $DIR/issue-62973.rs:6:25
   |
LL | fn p() { match s { v, E { [) {) }
   |                -      - ^ expected one of `,`, `:`, or `}`
   |                |      |
   |                |      help: try naming a field: `E:`
   |                while parsing this struct

error: struct literals are not allowed here
  --> $DIR/issue-62973.rs:6:16
   |
LL |   fn p() { match s { v, E { [) {) }
   |  ________________^
LL | |
LL | |
   | |_^
   |
help: surround the struct literal with parentheses
   |
LL ~ fn p() { match (s { v, E { [) {) }
LL | 
LL ~ )
   |

error: expected one of `.`, `?`, `{`, or an operator, found `}`
  --> $DIR/issue-62973.rs:8:2
   |
LL | fn p() { match s { v, E { [) {) }
   |          ----- while parsing this `match` expression
LL |
LL |
   |  ^ expected one of `.`, `?`, `{`, or an operator

error: aborting due to 5 previous errors

